src/slider.jsx

```diff
@@ -40,6 +38,122 @@ const valueInRangePropType = (props, propName, componentName) => {
   }
 };
 
+const getStyles = (props, state) => {
+  const {
+    slider,
+  } = state.muiTheme;
+
+  const fillGutter = slider.handleSize / 2;
+  const disabledGutter = slider.trackSize + slider.handleSizeDisabled / 2;
+  const calcDisabledSpacing = props.disabled ? ` - ${disabledGutter}px` : '';
+
+  const styles = {
+    root: {
+      touchCallout: 'none',
+      userSelect: 'none',
+      cursor: 'default',
+      height: slider.handleSizeActive,
+      position: 'relative',
+      marginTop: 24,
+      marginBottom: 48,
+    },
+    track: {
+      position: 'absolute',
+      top: (slider.handleSizeActive - slider.trackSize) / 2,
+      left: 0,
+      width: '100%',
+      height: slider.trackSize,
+    },
+    filledAndRemaining: {
+      position: 'absolute',
+      top: 0,
+      height: '100%',
+      transition: Transitions.easeOut(null, 'margin'),
+    },
+    handle: {
+      boxSizing: 'border-box',
+      position: 'absolute',
+      cursor: 'pointer',
+      pointerEvents: 'inherit',
+      top: 0,
+      left: '0%',
+      zIndex: 1,
+      margin: `${(slider.trackSize / 2)}px 0 0 0`,
+      width: slider.handleSize,
+      height: slider.handleSize,
+      backgroundColor: slider.selectionColor,
+      backgroundClip: 'padding-box',
+      border: '0px solid transparent',
+      borderRadius: '50%',
+      transform: 'translate(-50%, -50%)',
+      transition:
+        `${Transitions.easeOut('450ms', 'background')}, ${
+        Transitions.easeOut('450ms', 'border-color')}, ${
+        Transitions.easeOut('450ms', 'width')}, ${
+        Transitions.easeOut('450ms', 'height')}`,
+      overflow: 'visible',
+      outline: 'none',
+    },
+    handleWhenDisabled: {
```

Comment:
They were already in the code actually. I will try hard to change that, but it will take some time because there are a lot of cases that need to be handled (pun intended :smile:)